spoke-pm2.config.js.template

```diff
@@ -9,9 +9,12 @@ const env_production = {
   ASSETS_MAP_FILE:'assets.json',
   CAMPAIGN_ID:'campaign-id-hash',
   DB_HOST:'127.0.0.1',
-  DB_PORT:28015,
+  DB_PORT:5432,
   DB_NAME:'spokedev',
+  DB_USER:'spoke',
+  DB_PASSWORD:'spoke',
   DB_KEY:'',
+  DB_TYPE:'',
```

Comment:
Do we want to default to PostgreSQL for dev? I was thinking this should be blank by default for SQLite dev.